uint8 msgType
uint8 uid
float64 field1
float64 field2
float64 field3

